### DISCUSSION TITLE
q-option-group Select all

### DISCUSSION THREAD

**Discussion by akashramtohul4** [quasarframework/quasar#10147] [General - Components / Directives / etc] [answered]
Hello friends 

Am try to build a q-option-group which i can select all checkbox's at once and unselect all.
Please find the sample code below, can someone please help

https://codepen.io/akashramtohul4/pen/GRmvVEm?editors=1010
Thanks

**Comment by pdanpdan** [comment, marked as answer]
https://codepen.io/pdanpdan/pen/MWmvNQx?editors=1010

**Comment by akashramtohul4** [comment]
Thanks mate 👍
